analysis/src/ModuleMapper.cxx

```diff
@@ -1,514 +1,545 @@
-    sw_to_string["ly11_m0"] = "L6T_axial_ele" ;
-    sw_to_string["ly11_m1"] = "L6B_stereo_ele";
-    sw_to_string["ly11_m2"] = "L6T_axial_pos" ;
-    sw_to_string["ly11_m3"] = "L6B_stereo_pos";
-    sw_to_string["ly12_m0"] = "L6T_stereo_ele";
-    sw_to_string["ly12_m1"] = "L6B_axial_ele" ;
-    sw_to_string["ly12_m2"] = "L6T_stereo_pos";
-    sw_to_string["ly12_m3"] = "L6B_axial_pos" ;
-    
-    
-    string_to_sw["L1T_axial"     ] = "ly1_m0" ;
-    string_to_sw["L1B_stereo"    ] = "ly1_m1" ;
-    string_to_sw["L1T_stereo"    ] = "ly2_m0" ;
-    string_to_sw["L1B_axial"     ] = "ly2_m1" ;
-    string_to_sw["L2T_axial"     ] = "ly3_m0" ;
-    string_to_sw["L2B_stereo"    ] = "ly3_m1" ;
-    string_to_sw["L2T_stereo"    ] = "ly4_m0" ;
-    string_to_sw["L2B_axial"     ] = "ly4_m1" ;
-    string_to_sw["L3T_axial"     ] = "ly5_m0" ;
-    string_to_sw["L3B_stereo"    ] = "ly5_m1" ;
-    string_to_sw["L3T_stereo"    ] = "ly6_m0" ;
-    string_to_sw["L3B_axial"     ] = "ly6_m1" ;
-    string_to_sw["L4T_axial_ele" ] = "ly7_m0" ;
-    string_to_sw["L4B_stereo_ele"] = "ly7_m1" ;
-    string_to_sw["L4T_axial_pos" ] = "ly7_m2" ;
-    string_to_sw["L4B_stereo_pos"] = "ly7_m3" ;
-    string_to_sw["L4T_stereo_ele"] = "ly8_m0" ;
-    string_to_sw["L4B_axial_ele" ] = "ly8_m1" ;
-    string_to_sw["L4T_stereo_pos"] = "ly8_m2" ;
-    string_to_sw["L4B_axial_pos" ] = "ly8_m3" ;
-    string_to_sw["L5T_axial_ele" ] = "ly9_m0" ;
-    string_to_sw["L5B_stereo_ele"] = "ly9_m1" ;
-    string_to_sw["L5T_axial_pos" ] = "ly9_m2" ;
-    string_to_sw["L5B_stereo_pos"] = "ly9_m3" ;
-    string_to_sw["L5T_stereo_ele"] = "ly10_m0";
-    string_to_sw["L5B_axial_ele" ] = "ly10_m1";
-    string_to_sw["L5T_stereo_pos"] = "ly10_m2";
-    string_to_sw["L5B_axial_pos" ] = "ly10_m3";
-    string_to_sw["L6T_axial_ele" ] = "ly11_m0";
-    string_to_sw["L6B_stereo_ele"] = "ly11_m1";
-    string_to_sw["L6T_axial_pos" ] = "ly11_m2";
-    string_to_sw["L6B_stereo_pos"] = "ly11_m3";
-    string_to_sw["L6T_stereo_ele"] = "ly12_m0";
-    string_to_sw["L6B_axial_ele" ] = "ly12_m1";
-    string_to_sw["L6T_stereo_pos"] = "ly12_m2";
-    string_to_sw["L6B_axial_pos" ] = "ly12_m3";
-  }
-  else 
-    {std::cout<<"ERROR: Module Mapper cannot be setup for this year "<<year<<std::endl;}
-} //constructor
+    std::cout << "[ModuleMapper] Initialized" << std::endl;
+    if (year_ == 2019) {
+
+        //TODO Insert the correct mapping!!
+        //Cameron mapping = > This should be the correct mapping
+        //https://docs.google.com/spreadsheets/d/1sVtYEppxKlwJvpniemKf6EtFqK8qN0ZnDRExR_eqqZw/edit#gid=1695271832
+        //hw_to_sw["F3H0"] = "ly7_m1";
+        //hw_to_sw["F3H1"] = "ly8_m1";
+        //hw_to_sw["F3H2"] = "ly5_m1";
+        //hw_to_sw["F3H3"] = "ly6_m1";
+
+        //This is Omar's mapping, which is what is in the DB
+        //WARNING: Might be wrong.
+
+        std::cout<<"WARNING: ModuleMapper: using a mapping that might be wrong"<<std::endl;
+
```

Comment:
* Can we check that it is correct before we commit this code?
* Can we invert the one table, instead of entering the whole list twice? (and then again for the Python code.)